design/media_types.go

```diff
@@ -31,3 +31,66 @@ var AuthToken = MediaType("application/vnd.authtoken+json", func() {
 		Attribute("token")
 	})
 })
+
+var WorkItem = MediaType("application/vnd.workitem+json", func() {
+	TypeName("WorkItem")
+	Description("ALM Work Item")
+	Attribute("id", String, "unique id per installation")
```

Comment:
Federation can be done by id= <instanceid>:<localid>. Unless you want bug id's to be stable when moving to a different repository. Anyway, I think this is a topic for a larger discussion on what the requirements are in this respect.